### PR TITLE
translate reference-dom-elements

### PR DESCRIPTION
**Pull request by galnir**

### CODE REVIEW

File: content/docs/reference-dom-elements.md [hunk 1]
```diff
@@ -77,66 +77,66 @@ function HelloWorldComponent() {
...
 };
 
 function ComponentWithTransition() {
-  return <div style={divStyle}>This should work cross-browser</div>;
+  return <div style={divStyle}>זה צריך לעבוד בין דפדפנים</div>;
 }
 ```
 
-Style keys are camelCased in order to be consistent with accessing the properties on DOM nodes from JS (e.g. `node.style.backgroundImage`). Vendor prefixes [other than `ms`](https://www.andismith.com/blogs/2012/02/modernizr-prefixed/) should begin with a capital letter. This is why `WebkitTransition` has an uppercase "W".
+Style keys הם camelCased על מנת להיות עקביים עם גישה לproperties על הDOM nodes מJavaScript( לדוגמה `node.style.backgroundImage`). ספקי prefixes  [ששונים מ `ms`](https://www.andismith.com/blogs/2012/02/modernizr-prefixed/) צריכים להתחיל עם אות גדולה. לכן `WebkitTransition` מתחיל עם אות גדולה.
```
Comment (galtalmor) [inline content/docs/reference-dom-elements.md]:
```suggestion
מפתחות עיצוב הם camelCased על מנת להיות עקביים עם גישה למאפיינים על צמתי ה-DOM מ-JavaScript (לדוגמה `node.style.backgroundImage`). קדימויות של ספקים [ששונים מ-`ms`](https://www.andismith.com/blogs/2012/02/modernizr-prefixed/) צריכים להתחיל עם אות גדולה. זו הסיבה ש-`WebkitTransition` מתחיל עם אות גדולה.
```

File: content/docs/reference-dom-elements.md [hunk 2]
```diff
@@ -77,66 +77,66 @@ function HelloWorldComponent() {
...
 
-React will automatically append a "px" suffix to certain numeric inline style properties. If you want to use units other than "px", specify the value as a string with the desired unit. For example:
+React תוסיף באופן אוטומטי סיומת  "px" לכמה מאפייני סגנון. אם אתה רוצה להשתמש ביחידות מידה אחרות חוץ מ"px", נקוב בערך כמחרוזת עם יחידת המידה הרצויה. לדוגמה:
```
Comment (galtalmor) [inline content/docs/reference-dom-elements.md]:
```suggestion
React תוסיף באופן אוטומטי סיומת  "px" לכמה מאפייני סגנון. אם אתה רוצה להשתמש ביחידות מידה אחרות חוץ מ-"px", נקוב בערך כמחרוזת עם יחידת המידה הרצויה. לדוגמה:
```

File: content/docs/reference-dom-elements.md [hunk 3]
```diff
@@ -77,66 +77,66 @@ function HelloWorldComponent() {
...
-// Result style: '10%'
+// תוצאה: '10%'
 <div style={{ height: '10%' }}>
-  Hello World!
+  שלום עולם!
 </div>
 ```
 
-Not all style properties are converted to pixel strings though. Certain ones remain unitless (eg `zoom`, `order`, `flex`). A complete list of unitless properties can be seen [here](https://github.com/facebook/react/blob/4131af3e4bf52f3a003537ec95a1655147c81270/src/renderers/dom/shared/CSSProperty.js#L15-L59).
+לא כל הstyle properties מומרים למחרוזות פיקסלים. כמה מהם נשארים ללא יחידת מידה( לדוגמה `zoom`, `order`, `flex`). רשימה מלאה נמצאת [כאן](https://github.com/facebook/react/blob/4131af3e4bf52f3a003537ec95a1655147c81270/src/renderers/dom/shared/CSSProperty.js#L15-L59).
```
Comment (galtalmor) [inline content/docs/reference-dom-elements.md]:
```suggestion
לא כל מאפייני העיצוב מומרים למחרוזות פיקסלים. כמה מהם נשארים ללא יחידת מידה (לדוגמה `zoom`, `order`, `flex`). רשימה מלאה של מאפיינים ללא יחידות מידה נמצאת [כאן](https://github.com/facebook/react/blob/4131af3e4bf52f3a003537ec95a1655147c81270/src/renderers/dom/shared/CSSProperty.js#L15-L59).
```

File: content/docs/reference-dom-elements.md [hunk 4]
```diff
@@ -77,66 +77,66 @@ function HelloWorldComponent() {
...
 
 ### suppressContentEditableWarning {#suppresscontenteditablewarning}
 
-Normally, there is a warning when an element with children is also marked as `contentEditable`, because it won't work. This attribute suppresses that warning. Don't use this unless you are building a library like [Draft.js](https://facebook.github.io/draft-js/) that manages `contentEditable` manually.
+בדרך כלל, ישנה אזהרה כשאלמנט עם ילדים מסומן כ`contentEditable`, בגלל שהוא לא יעבוד. תכונה זו עוצרת את האזהרה. לא מומלץ להשתמש בזה אלא אם אתה בונה ספרייה כמו [Draft.js](https://facebook.github.io/draft-js/) שמנהלת `contentEditable` באופן ידני.
```
Comment (galtalmor) [inline content/docs/reference-dom-elements.md]:
```suggestion
בדרך כלל, ישנה אזהרה כשאלמנט עם ילדים מסומן כ-`contentEditable`, בגלל שזה לא יעבוד. תכונה זו עוצרת את האזהרה. אל תשתמש בזה אלא אם אתה בונה ספרייה כמו [Draft.js](https://facebook.github.io/draft-js/) שמנהלת `contentEditable` באופן ידני.
```

File: content/docs/reference-dom-elements.md [hunk 5]
```diff
@@ -77,66 +77,66 @@ function HelloWorldComponent() {
...
 
 ### suppressHydrationWarning {#suppresshydrationwarning}
 
-If you use server-side React rendering, normally there is a warning when the server and the client render different content. However, in some rare cases, it is very hard or impossible to guarantee an exact match. For example, timestamps are expected to differ on the server and on the client.
+אם אתה משתמש ברינדור server-side בReact, בדרך כלל יש אזהרה כשהשרת והלקוח מרנדרים תוכן שונה. לעומת זאת, במקרים נדירים, זה קשה מאוד עד בלתי אפשרי להבטיח תוצאה זהה. לדוגמה, חותמות זמן אמורות להיות שונות בשרת ובלקוח.
```
Comment (galtalmor) [inline content/docs/reference-dom-elements.md]:
```suggestion
אם אתה משתמש ברינדור בצד-שרת ב-React, בדרך כלל יש אזהרה כשהשרת והלקוח מרנדרים תוכן שונה. לעומת זאת, במקרים נדירים, זה קשה מאוד עד בלתי אפשרי להבטיח תוצאה זהה. לדוגמה, חותמות זמן אמורות להיות שונות בשרת ובלקוח.
```

File: content/docs/reference-dom-elements.md [hunk 6]
```diff
@@ -77,66 +77,66 @@ function HelloWorldComponent() {
...
 
-If you set `suppressHydrationWarning` to `true`, React will not warn you about mismatches in the attributes and the content of that element. It only works one level deep, and is intended to be used as an escape hatch. Don't overuse it. You can read more about hydration in the [`ReactDOM.hydrate()` documentation](/docs/react-dom.html#hydrate).
+אם אתה קובע את `suppressHydrationWarning` כtrue, React לא תזהיר אותך לגבי אי התאמה בתכונות ובתוכן של האלמנט. דבר זה עובד בעומק של אלמנט אחד בלבד, ומיועד לשימוש כדרך מוצא. לא מומלץ להשתמש בזה יותר מדי. ניתן לקרוא עוד על hydration ב  [`ReactDOM.hydrate()` documentation](/docs/react-dom.html#hydrate).
```
Comment (galtalmor) [inline content/docs/reference-dom-elements.md]:
```suggestion
אם אתה קובע את `suppressHydrationWarning` כ-`true`, React לא תזהיר אותך לגבי אי התאמה בתכונות ובתוכן של האלמנט. דבר זה עובד בעומק של רמה אחת בלבד, ומיועד לשימוש כפתח מילוט. אל תשתמש בזה יותר מדי. ניתן לקרוא עוד על hydration ב-[`ReactDOM.hydrate()` documentation](/docs/react-dom.html#hydrate).
```

File: content/docs/reference-dom-elements.md [hunk 7]
```diff
@@ -77,66 +77,66 @@ function HelloWorldComponent() {
...
 
 ### value {#value}
 
-The `value` attribute is supported by `<input>` and `<textarea>` components. You can use it to set the value of the component. This is useful for building controlled components. `defaultValue` is the uncontrolled equivalent, which sets the value of the component when it is first mounted.
+תכונת ה`value` נתמכת על ידי הקומפוננטות `<input>` `<textarea>`. ניתן להשתמש בזה לקבוע את ערך הקומפוננטה. זה שימושי לבניית קומפוננטות נשלטות. `defaultValue` הוא המקביל הלא נשלט, שקובע את ערך הקומפוננטה כשהיא לראשונה mounted.
```
Comment (galtalmor) [inline content/docs/reference-dom-elements.md]:
```suggestion
תכונת ה-`value` נתמכת על ידי הקומפוננטות `<input>` ו-`<textarea>`. ניתן להשתמש בה כדי לקבוע את ערך הקומפוננטה. זה שימושי לבניית קומפוננטות נשלטות. `defaultValue` הוא המקביל הבלתי נשלט, שקובע את ערך הקומפוננטה כשהיא mounted לראשונה.
```

File: content/docs/reference-dom-elements.md [hunk 8]
```diff
@@ -77,66 +77,66 @@ function HelloWorldComponent() {
...
 
-## All Supported HTML Attributes {#all-supported-html-attributes}
+## כל תכונות הHTML הנתמכות: {#all-supported-html-attributes}
```
Comment (galtalmor) [inline content/docs/reference-dom-elements.md]:
```suggestion
## כל תכונות ה-HTML הנתמכות: {#all-supported-html-attributes}
```

File: content/docs/reference-dom-elements.md [hunk 9]
```diff
@@ -77,66 +77,66 @@ function HelloWorldComponent() {
...
 
-As of React 16, any standard [or custom](/blog/2017/09/08/dom-attributes-in-react-16.html) DOM attributes are fully supported.
+החל מReact 16, כל תכונת DOM סטנדרטית או מותאמת אישית, נתמכת באופן מלא.
```
Comment (galtalmor) [inline content/docs/reference-dom-elements.md]:
```suggestion
החל מ-React 16, כל תכונת DOM סטנדרטית או [מותאמת אישית](/blog/2017/09/08/dom-attributes-in-react-16.html), נתמכת באופן מלא.
```

File: content/docs/reference-dom-elements.md [hunk 10]
```diff
@@ -77,66 +77,66 @@ function HelloWorldComponent() {
...
 
-React has always provided a JavaScript-centric API to the DOM. Since React components often take both custom and DOM-related props, React uses the `camelCase` convention just like the DOM APIs:
+React תמיד סיפקה ממשק משתמש ממוקד JavaScript לDOM. מכיוון שקומפוננטות React מקבלות props מותאמות אישית וגם כאלה שקשורות לDOM, React משתמש במוסכמת ה`camelCase` כמו ממשקי משתמש של הDOM:
```
Comment (galtalmor) [inline content/docs/reference-dom-elements.md]:
```suggestion
React תמיד סיפקה ממשק משתמש ממוקד JavaScript ל-DOM. מכיוון שקומפוננטות React בדרך כלל מקבלות גם props מותאמות אישית וגם כאלה שקשורות ל-DOM, React משתמשת במוסכמת ה-`camelCase` בדיוק כמו ממשקי המשתמש של ה-DOM:
```

File: content/docs/reference-dom-elements.md [hunk 11]
```diff
@@ -77,66 +77,66 @@ function HelloWorldComponent() {
...
-<div tabIndex="-1" />      // Just like node.tabIndex DOM API
-<div className="Button" /> // Just like node.className DOM API
-<input readOnly={true} />  // Just like node.readOnly DOM API
+<div tabIndex="-1" />      // כמו node.tabIndex DOM API
+<div className="Button" /> // כמו node.className DOM API
+<input readOnly={true} />  // כמו node.readOnly DOM API
 ```
 
-These props work similarly to the corresponding HTML attributes, with the exception of the special cases documented above.
+הprops הללו עובדים באופן דומה לתכונות הHTML המקבילות, עם חריגה של מקרים מיוחדים שנמצאים לעיל.
```
Comment (galtalmor) [inline content/docs/reference-dom-elements.md]:
```suggestion
ה-props הללו עובדים באופן דומה לתכונות ה-HTML המקבילות, עם הוצאה מן הכלל של המקרים המיוחדים המתועדים לעיל.
```

File: content/docs/reference-dom-elements.md [hunk 12]
```diff
@@ -77,66 +77,66 @@ function HelloWorldComponent() {
...
 
-Some of the DOM attributes supported by React include:
+חלק מתכונות הDOM הנתמכות על ידי React כוללות:
```
Comment (galtalmor) [inline content/docs/reference-dom-elements.md]:
```suggestion
חלק מתכונות ה-DOM הנתמכות על ידי React כוללות:
```

File: content/docs/reference-dom-elements.md [hunk 13]
```diff
@@ -154,7 +154,7 @@ sizes span spellCheck src srcDoc srcLang srcSet start step style summary
 tabIndex target title type useMap value width wmode wrap
 ```
 
-Similarly, all SVG attributes are fully supported:
+באופן דומה, כל תכונות הSVG נתמכות באופן מלא:
```
Comment (galtalmor) [inline content/docs/reference-dom-elements.md]:
```suggestion
באופן דומה, כל תכונות ה-SVG נתמכות באופן מלא:
```

**Comment by galnir** [comment]
Thanks @galtalmor 👍

**Comment by galtalmor** [comment]
@galnir Sure!
You skipped some comments that were collapsed. Also, you can batch multiple changes in the files tab to merge at once

**Pull request merged**